client/coniksclient/internal/cmd/lookup.go

```diff
@@ -22,6 +22,8 @@ var lookupCmd = &cobra.Command{
 			cmd.Usage()
 			return
 		}
+		bindings := loadKeys(conf.KeyStoragePath)
```

Comment:
I'm wondering if `bindings` should be part of `cc`? I think it makes total sense to (re-)load the keys whenever the client does a lookup as it's implemented right now. But I'm also wondering if there would be some advantage to keeping one global `bindings` map as part of the `cc` state instead.